docs/IE_DG/protecting_model_guide.md

```diff
@@ -43,6 +43,9 @@ Extensions](https://software.intel.com/en-us/sgx).
 Use `InferenceEngine::Core::ReadNetwork()` to set model representations and
 weights respectively.
 
+Note: Reading a ONNX model which uses weights saved in external files
+is not supported by `ReadNetwork(const std::string& model, const Blob::CPtr& weights)` overload.
```

Comment:
Please add information that weights should be empty or equal nullptr.

Reply:
done